boards/st/stm32h7s78_dk/stm32h7s78_dk.yaml

```diff
@@ -4,7 +4,7 @@ type: mcu
 arch: arm
 toolchain:
   - zephyr
-ram: 640
+ram: 128
```

Comment:
I'll likely drop this change that is addressed in https://github.com/zephyrproject-rtos/zephyr/pull/97992.